client/src/components/landing/components/LandingTop.js

```diff
@@ -30,6 +30,7 @@ function LandingTop({ page }) {
           </h1>
           <p className='big-heading'>{t('landing.big-heading-2')}</p>
           <p className='big-heading'>{t('landing.big-heading-3')}</p>
+          <p className='big-heading'>{t('landing.big-heading-4')}</p>
```

Comment:
And the key is used here, but is not replacing any existing un-translated text. So I am confused about the addition of text to the client. Why is this being added?